{"name":"verifyUnSuccessfulRegisteration","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Register] but found [My Account]","trace":"java.lang.AssertionError: expected [Register] but found [My Account]\r\n\tat org.testng.Assert.fail(Assert.java:97)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:575)\r\n\tat org.testng.Assert.assertEquals(Assert.java:585)\r\n\tat completeScenarios.NPwdNnchar.verifyUnSuccessfulRegisteration(NPwdNnchar.java:67)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:766)\r\n\tat org.testng.TestRunner.run(TestRunner.java:587)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1109)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1039)\r\n\tat org.testng.TestNG.run(TestNG.java:1007)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"Test all fields are valid but the password does not contain any char. 3. Verify unsuccessful registeration...","steps":[],"attachments":[{"name":"saveFailureScreenShot","source":"9e9d6828-3741-41ba-9ac1-60b34bcb0332-attachment"},{"name":"verifyUnSuccessfulRegisteration failed and screenshot taken!","source":"53627196-a23f-4edc-9ceb-23679c351a7e-attachment","type":"text/plain"}],"parameters":[],"start":1624744100950,"stop":1624744101062,"uuid":"214b4219-8219-4214-a192-b8824aac94b8","historyId":"97e8da80e16c264fecb74f6365860e1c","fullName":"completeScenarios.NPwdNnchar.verifyUnSuccessfulRegisteration","labels":[{"name":"package","value":"completeScenarios.NPwdNnchar"},{"name":"testClass","value":"completeScenarios.NPwdNnchar"},{"name":"testMethod","value":"verifyUnSuccessfulRegisteration"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"completeScenarios.NPwdNnchar"},{"name":"host","value":"Hagar-Elsayed"},{"name":"thread","value":"24336@Hagar-Elsayed.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}